doc/procedures/dockermacm1.rst

```diff
@@ -0,0 +1,52 @@
+################################
+Docker for development on Mac M1
+################################
+
+Most of these instructions are valid for any docker based development in but the M1 has a couple of special requirements. 
+
+Install and setup Docker
+========================
+Install docker by e..g downloading the install from `the docker website <https://www.docker.com/products/docker-desktop/>`_
+
+
+Once you have docker desktop running open the settings (Cmd ,)
+
+- In the General settings make sure 'VirtioFS' is selected. 
+
+.. figure:: ../images//DockerSettingG.png
+   :name: fig-dockerg
+   :alt: Docker General Settings
+
+- In the Features in Development settings select 'Use Rosetta for X86/amd64 emulation on Apple Silicon'
+
+.. figure:: ../images//DockerSettingD.png
+   :name: fig-dockerd
+   :alt: Docker Development Settings
+
+
+Running the container
+=====================
+
+The `develop` tag indicates the most recent container that should be used for development.
+The development container is `develop-env`.
```

Comment:
What about stable containers - cycles? That's handy when you need containers running on summit. https://confluence.lsstcorp.org/display/LSSTCOM/Software+Upgrades lists cycles.

https://hub.docker.com/r/lsstts/develop-env/tags lists develop-env tags on Dockerhub. However, as the images were moved to own server, one now needs to retrieve cCCCC.VVV, where CCCC is cycle number (0029) and VVV is version (000 for example). So to get proper c0029, I use:

ts-dockerhub.lsst.org/develop-env:c0029.000

Reply:
ok put a version of this in the text

Reply:
I only see a c0028.004